### DISCUSSION TITLE
🙏 Moving the grid foreground and background

### DISCUSSION THREAD

**Discussion by Cicorione** [speedata/publisher#511] [Q&A]
Hi Patrick,

during my previous tests I used a background image and I noticed the image background was covering the grid, I wonder if you can add an option to move the grid foreground like:

```xml
<Trace grid="yes" foreground="yes"/> # default would be "no"
```

Thanks... 🙏

**Comment by pgundlach** [comment]
I close this discussion as I have created issue #512 from it.
